internal/node/_node_bin/node

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Revert this file?

Reply:
This happened automatically when I was trying to commit the changes.

Reply:
Sure, I'll try to revert it.